src/System.Private.CoreLib/src/System/Runtime/CompilerServices/jithelpers.cs

```diff
@@ -102,7 +102,17 @@ internal static StackCrawlMarkHandle GetStackCrawlMarkHandle(ref StackCrawlMark
         internal static ref byte GetRawData(this object obj) =>
             ref Unsafe.As<RawData>(obj).Data;
 
-        internal static ref byte GetRawSzArrayData(this Array array) =>
-            ref Unsafe.As<RawSzArrayData>(array).Data;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
We have API proposal for this in https://github.com/dotnet/corefx/issues/33706 . If you would like to use this method in more places, we should add it to RuntimeHelpers where it is likely to be eventually and do not make it an extension method.